### ISSUE TITLE
Error during installation of Iterable SDK (react-native versions conflict)

### ISSUE THREAD

**Issue by Luckygirlllll** [Iterable/react-native-sdk#117]
I'm getting an error when trying to add Iterable SDK to my project.

`npm install @iterable/react-native-sdk`

```
 Found: react-native@0.63.4
npm ERR! node_modules/react-native
npm ERR!   react-native@"0.63.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native@"^0.62.2" from @iterable/react-native-sdk@1.0.25
npm ERR! node_modules/@iterable/react-native-sdk
npm ERR!   @iterable/react-native-sdk@"*" from the root project
```

Does this SDK support react-native v0.63.4 ?

**Comment by roninopf** [comment]
Hi @Luckygirlllll, this is an error on our part, we do support it, I'll be fixing this shortly.

**Comment by roninopf** [comment]
We just released version 1.0.26 which now allows any version of `react` and `react-native` to be a peer dependent. This should fix this. Please feel free to open another issue through your CSM (it'll be faster) if one arises. Thanks!
